completions/evince

```diff
@@ -25,7 +25,7 @@ _evince()
         return
     fi
 
-    _filedir '@(@(?(e)ps|?(E)PS|[pf]df|[PF]DF|dvi|DVI)?(.gz|.GZ|.bz2|.BZ2|.xz|.XZ)|cb[rz]|djv?(u)|tif?(f))'
+    _filedir '@(@(?(e)ps|?(E)PS|[pf]df|[PF]DF|dvi|DVI)?(.gz|.GZ|.bz2|.BZ2|.xz|.XZ)|cb[7rtz]|djv?(u)|tif?(f)||?(o)xps)'
```

Comment:
I think the `|`'s before `?(o)xps` are duplicate.

Reply:
Sigh, thanks. Fixed in a79746c62917c25a17b4711e3d8527f2b67b0a5c